.github/workflows/deploy.yml

```diff
@@ -0,0 +1,25 @@
+name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+
+on: push
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Build distribution
+      run: |
+        python setup.py sdist
+    - name: Publish distribution 📦 to Test PyPI
+      uses: pypa/gh-action-pypi-publish@master
+      with:
+        password: ${{ secrets.TEST_PYPI_API_TOKEN }}
```

Comment:
I am wondering who's username it takes here if we only pass the password? In linopy/atlite we added user name and password as secrets (which are then both passed here)

Reply:
Yes, here it's my tokens stored as secret (although I still need to be added as PyPSA maintainer on Test PyPI which is why the CI currently fails)

Reply:
Ah okay, which means that only you and tom can do the tagged releases (which is totally be with me) since it takes the GH username of the one who pushes, right?

Reply:
No, I think everyone with push rights to https://github.com/pypsa/pypsa can tag a new release.